mini-bank/create-account.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Create your account</title>
+    <link rel="stylesheet" href="../helpers/css/components.css">
+    <link rel="stylesheet" href="mini-bank.css">
+</head>
+<body>
+    <div class="container-fixed create-container">
```

Comment:
`create-container` class should be wrapped inside `container-fixed`, because `container-fixed` styles can conflict with `create-container` styles.
Do this instead:
```
...
<div class="container-fixed">
    <div class="create-container">
         ...
    </div>
</div>
...
```

Also, since you're renaming this file to `landing.html`, you should change all associated namings in the file accordingly. So `create-container` class should be `landing-container`.